README.md

```diff
@@ -42,7 +42,7 @@ You then need to load the plugin. In `boostrap.php`, something like:
 For a complete setup, add the following to your `App\Controller\AppController`:
 
 ```php
-public $components = ['BootstrapUI.Flash'];
+public $theme = 'BootstrapUI';
```

Comment:
It's probably better to bake them into the app, rather than add the overhead of a theme for all views, that are going to be used for flash elements only - but there aren't any bake templates yet (I do that).